content/en/docs/project-administration/project-network-isolation.md

```diff
@@ -1,133 +1,181 @@
 ---
 title: "Project Network Isolation"
-keywords: 'KubeSphere, kubernetes, Calico, Network Policy'
+keywords: 'KubeSphere, Kubernetes, Calico, Network Policy'
 description: 'Project Network Isolation'
-
 linkTitle: "Project Network Isolation"
 weight: 13300
 ---
 
-KubeSphere project network isolation lets project administrators enforce which network traffic is allowed using rules.
+KubeSphere project network isolation lets project administrators enforce which network traffic is allowed using different rules. This tutorial demonstrates how to enable network isolation among projects and set rules to control network traffic.
 
 ## Prerequisites
 
-- You have already enabled Network Policy. Please refer to [network-policy](../../pluggable-components/network-policy) if it is not ready yet.
-- Use an account of the `admin` role at the project level. For example, use the account `project-admin` created in [Create Workspace, Project, Account and Role](../../quick-start/create-workspace-and-project/).
+- You have already enabled [Network Policies](../../pluggable-components/network-policy/).
+- You must have an available project and an account of the `admin` role (`project-admin`) at the project level. For more information, see [Create Workspace, Project, Account and Role](../../quick-start/create-workspace-and-project/).
 
 {{< notice note >}}
-For the implementation of the Network Policy, you can refer to [kubesphere-network-policy](https://github.com/kubesphere/community/blob/master/sig-network/concepts-and-designs/kubesphere-network-policy.md).
+
+For the implementation of the Network Policy, you can refer to [KubeSphere NetworkPolicy](https://github.com/kubesphere/community/blob/master/sig-network/concepts-and-designs/kubesphere-network-policy.md).
```

Comment:
Is there a versioned branch design? If yes, it is better to use it instead of master branch